wgpu-core/src/instance.rs

```diff
@@ -217,6 +218,32 @@ impl AdapterInfo {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+/// Error when requesting a device from the adaptor
+pub enum RequestDeviceError {
+    /// Unsupported feature extension was requested
+    UnsupportedFeature(wgt::Features),
+    /// Adapter doesn't support the amount of requested bind groups
+    LimitsExceeded(u32),
```

Comment:
we could do many things here: provide the adapter Limits, provide both limits, or none of them.
We could also provide the "difference" of device requested limits minus the adapter, just like we do with features.
Just `u32` is not appropriate

Reply:
Okay, I can remove it for now. I manged to mess up my branch by not rebaseing before squashing so that's why all the force pushes :sweat_smile: